dev-support/bin/hadoop.sh

```diff
@@ -614,7 +615,7 @@ function shadedclient_rebuild
   echo_and_redirect "${logfile}" \
     "${MAVEN}" "${MAVEN_ARGS[@]}" verify -fae --batch-mode -am \
       "${modules[@]}" \
-      -DskipShade -Dtest=NoUnitTests -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true \
+      -DskipShade -Dsurefire.failIfNoSpecifiedTests=false -Dtest=NoUnitTests -Dmaven.javadoc.skip=true -Dcheckstyle.skip=true \
```

Comment:
I carefully reviewed the build report, and `-Dsurefire.failIfNoSpecifiedTests=false` is necessary because we have some modules without JUnit tests. We use `-Dtest=NoUnitTests` to handle this, which can cause an error.

Reply:
can we just make this the default in our pom definitions? I disable tests all the time to run failsafe ITests in the integration phase only.